addons/binding/org.openhab.binding.foobot/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="airboxlab" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="foobot">
+		<label>Foobot</label>
+		<description>
+        Provides a list of all foobots in your home.
+        In order to retrieve the list you must register have an account on foobot.io and you must get your API token from https://api.foobot.io/apidoc/index.html
+        </description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature" />
+			<channel id="humidity" typeId="humidity" />
+			<channel id="voc" typeId="voc" />
+			<channel id="co2" typeId="co2" />
+			<channel id="pm" typeId="pm" />
+			<channel id="gpi" typeId="gpi" />
+
+		</channels>
+
+		<config-description>
+			<parameter name="apikey" type="text" required="true">
+				<context>password</context>
+				<label>API Key</label>
+				<description>You can request your API Key from https://api.foobot.io/apidoc/index.html </description>
+			</parameter>
+
+			<parameter name="username" type="text" required="true">
+				<label>User Id</label>
+				<description>UserId of foobot account. Its the user Id that you use to login to the Foobot account.</description>
+			</parameter>
+
+			<parameter name="mac" type="text" required="true">
+				<label>Mac Address</label>
+				<description>MAC address of your device. You can find it in the Foobot App under settings.</description>
+			</parameter>
+
+			<parameter name="refresh" type="integer" min="5" required="false">
+				<label>Refresh interval</label>
+				<description>Specifies the refresh interval in minutes.</description>
+				<default>30</default>
+			</parameter>
+
+		</config-description>
+	</thing-type>
+
+	<channel-type id="temperature">
+		<item-type>Number</item-type>
+		<label>Temperature (°C)</label>
+		<description>Temperature in degree Celsius</description>
```

Comment:
As you don't mention the unit in the other channels, I wouldn't do so here too. Especially in the light of the new units support.